### ISSUE TITLE
[Bug]: Fix Alignment of Eye Button in Password Field on Login Page

### ISSUE THREAD

**Issue by ritiksingh-01** [mansiruhil13/Bobble-AI#196]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The eye button used to toggle password visibility on the login page is not properly aligned within the password field. This misalignment affects the overall visual consistency and usability of the login interface. A proper alignment will enhance user experience by ensuring that the eye button is easily accessible and visually appealing.

Additionally, I can redesign the login and sign-up pages to improve their functionality by integrating a Google login option. This enhancement will provide users with:

- **Faster Authentication:** Allowing users to sign in quickly with their Google accounts, reducing the time spent on the login process.
- **Enhanced Security:** Utilizing Google’s OAuth authentication ensures secure access to their accounts.
- **Convenience:** Eliminating the need for users to remember multiple passwords, leading to a smoother login experience.

These improvements will significantly enhance user satisfaction and streamline the authentication process.              

Kindly assign this issue to me under the GSSoC'24-Extd program.

Thank you for your consideration.

### Add Screenshots

![login page](https://github.com/user-attachments/assets/f7cd76e2-da90-4cf7-9679-1ce58575cded)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by mansiruhil13** [comment]
its' better, i think no need. on login page there is a need of ui improvement

**Comment by ritiksingh-01** [comment]
I understand that the login page requires UI improvements. I can enhance the design to make it more appealing and user-friendly, ensuring a smoother experience. Additionally, I will integrate a section for login and signup with Google, allowing users to access their accounts more efficiently and securely. These updates will significantly improve the overall usability of the login page.

**Comment by ritiksingh-01** [comment]
> its' better, i think no need. on login page there is a need of ui improvement

I understand that the login page requires UI improvements. I can enhance the design to make it more appealing and user-friendly, ensuring a smoother experience. Additionally, I will integrate a section for login and signup with Google, allowing users to access their accounts more efficiently and securely. These updates will significantly improve the overall usability of the login page.

**Comment by mansiruhil13** [comment]
you can work but ui ux of login already assigned.

**Comment by ritiksingh-01** [comment]
> you can work but ui ux of login already assigned.

I understand that the UI/UX improvements for the login page are already assigned to someone else. However, I believe I can contribute valuable enhancements as well. It would be great if both sets of changes could be considered, and the preferred options could be merged accordingly.

Please assign this issue to me so I can start working on it. Thank you!

**Comment by ritiksingh-01** [comment]
Hi, @mansiruhil13 

https://github.com/user-attachments/assets/0a13e3a3-5da4-4529-bc1a-ed32c5fe6fd2


I have completed the main UI/UX design, including the integration of other platform features. I kindly request you to review it at your earliest convenience. Please allow me to proceed with submitting a pull request once the review is complete.

**Comment by mansiruhil13** [comment]
yes you can

**Comment by mansiruhil13** [comment]
sorry but that already done, and ui improvements assigned recently

**Comment by mansiruhil13** [comment]
you can find and work on other issue
